.eslintrc.js

```diff
@@ -1,25 +1,109 @@
+const path = require('path');
+
 module.exports = {
-    env: {
-        browser: true,
-        es2021: true,
+  root: true,
+  env: {
+    browser: true,
+    es2021: true,
+    node: true,
+    'jest/globals': true
+  },
+  extends: [
+    'plugin:react/recommended',
+    'plugin:import/typescript',
```

Comment:
all configurations from Typescript should be in a typescript branch

Reply:
Please all configuration of typescript in other branch, by voting it was decided that the base template will be used with proptypes.